File: exercicios/2031/2031.py
def entrada():

    n = int(input())

    return(n)

def main():
    n = entrada()
    g1 = 'Jogador 1 venceu'
    g2 = 'Jogador 2 venceu'
    for i in range(n):
        j1 = input()
        j2 = input()

        if j1 == 'ataque' and j2 == 'pedra':
            print(g1)
        elif j1 == 'pedra' and j2 == 'ataque':
            print(g2)

        elif j1 == 'pedra' and j2 == 'papel':
            print(g1)
        elif j1 == 'papel' and j2 == 'pedra':
            print(g2)
        
        elif j1 == 'ataque' and j2 == 'papel':
            print(g1)
        elif j1 == 'papel' and j2 == 'ataque':
            print(g2)
        
        elif j1 == 'papel' and j2 == 'papel':
            print('Ambos venceram')
        
        elif j1 == 'pedra' and j2 == 'pedra':
            print('Sem ganhador')
        
        elif j1 == 'ataque' and j2 == 'ataque':
            print('Aniquilacao mutua')

        i+=1
main()

File: exercicios/JavaScript/2031/2031.js
const { parseArgs } = require("util");

var input = require("fs").readFileSync('stdin','utf-8');
var lines = input.split('\n');

var x = lines.shift();

var sg = "Sem ganhador";
var jg1 = "Jogador 1 venceu";
var jg2 = "Jogador 2 venceu";
var ag = "Ambos venceram";
var am = "Aniquilacao mutua";

for(let i=0; i<x; i++){
    var j = 0
    var op1 = ""
    var op2 = ""

    while(j<2){
        if(j==0){
            op1 = lines.shift().trim();
        }else if(j==1){
            op2 = lines.shift().trim();
        }
        j++
    }
    if(op1 == 'pedra' && op2 == 'pedra'){
        console.log(sg);
    }else if(op1 == 'papel' && op2 == 'papel'){
        console.log(ag)
    }else if(op1 == 'pedra' && op2 == 'pedra'){
        console.log(jg2);
    } else if(op1 == 'ataque' && op2 == 'papel'){
        console.log(jg1);
    } else if(op1 == 'ataque' && op2 == 'pedra'){
            console.log(jg1);
      }else if(op1 == 'pedra' && op2 == 'papel'){
            console.log(jg1);
      }else if(op1 == 'papel' && op2 == 'pedra'){
            console.log(jg2);
        }else if(op1 == 'papel' && op2 == 'ataque'){
            console.log(jg2);
        }else if(op1 == 'ataque' && op2 == 'papel'){
            console.log(jg1);
        }else if(op1 == 'ataque' && op2 == 'ataque'){
            console.log(am);
        }else if(op1 == 'pedra' && op2 == 'ataque'){
            console.log(jg2);}
}
    
